MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m

```diff
@@ -191,21 +189,4 @@ -(void) saveSpotifyData:(NSDictionary *) spotifyData{
          object:self];
     }];
 }
-
```

Comment:
Moved this function in the saveSpotifyData

Reply:
I guess you mean that you replaced saveSpotifyUserData with saveSpotifyData, based on [this](https://github.com/amanial1234/MetaU_Project/pull/16/files#r936235206)

Reply:
I didn't replace it but added it to the function so I can just have both calls in one function